task1/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Normally we just create a file per component. In this case it is okay but it is just a comment.